Review-08-Numerical-Integration-and-Differentiation.ipynb

```diff
@@ -194,6 +194,18 @@
     "(b) Degree of n-point Gaussian quadrature rule is $2n-1$."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 8.16. Newton-Cotes and Gaussian quadrature rules are both based on polynomial interpolation. (a) What specific property characterizes a Newton-Cotes rule for a given number of nodes?\n",
+    "(b)What specific property characterizes a Gaussian quadrature rule for a given number of nodes?\n",
+    "\n",
+    "(a) Newton-Cotes rules have equally spaced points along the interval.\n",
```

Comment:
You could also mention that unlike Gaussian quadrature rules, the weights are fixed, but perhaps that is implied by the fact the nodes are equally spaced.